components/builder-api/src/server/handlers/channel.rs

```diff
@@ -26,7 +26,7 @@ impl Handler<CreateChannel> for DbExecutor {
         match Channel::create(channel, self.get_conn()?.deref()) {
             Ok(channel) => Ok(channel),
             Err(DatabaseError(DatabaseErrorKind::UniqueViolation, _)) => {
-                Err(error::ErrorConflict("channel already Exists"))
+                Err(error::ErrorConflict("channel already exists"))
```

Comment:
Thanks for fixing that. I saw it in the logs the other day and I was kicking myself for typo-ing it